modules/effects/src/effects_resolver.ts

```diff
@@ -61,3 +50,19 @@ export function mergeEffects(
 
   return merge(...observables$);
 }
+
+/**
+ * @internal
+ */
+export function resubscribeInCaseOfError<T extends Action = Action>(
```

Comment:
No need to export it, right?

Reply:
Right, it also doesn't need the `@internal` comment